packages/ui/src/foundations/themes/index.ts

```diff
@@ -105,5 +105,6 @@ declare module '@mui/material/styles/createTheme' {
   }
 }
 
+export const inputMaxWidth = _inputMaxWidth;
```

Comment:
```suggestion
export { inputMaxWidth } from './light';
```

Can we just do this? I don't think we need to do the alias import 

In an ideal world `inputMaxWidth` shouldn't really exist. I wonder if we can integrate it into the theme

Reply:
Yea I agree, we could probably work to move this into the theme. Components that need it can reference it from there

Reply:
As discussed in the Staff Mtg, the `inputMaxWidth`-related refactoring will be completed in a separate ticket to be created by @jaalah-akamai